### DISCUSSION TITLE
New Installer feedback

### DISCUSSION THREAD

**Discussion by Annolago** [flybywiresim/aircraft#3151] [Ideas]
Hi

Just wanted to to congratulate your guys with the new installer. Nice layout, love the added possibility for engine variants in the future. not to say I cannot wait for the A380 project to be released.
However I will also say that you need to implement a nicer way to exit the programme. Only being able to end it by using the "X" option in the top right corner is normally what you do to an app that is hanging. 
It would be so much nicer to have an "Exit" button appear after the install is completed. That way no one is in doubt about when the programme is completed correctly.
Hope you will see the sense in this. 
Thank you for the hard work you all have put it to this project!

**Comment by robinottawa** [comment]
I have always had a "EXIT" button at the bottom of the installer. At least I think it's been there for some time.
EDIT: Oh, you said "new" installer. I think I didn't know there was one. Will check.....
EDIT2: I see the new installer. And there is no "exit" button. But it seems ok to me. You have a progress indicator. It's laid out like a web page and I would not personally expect to close it with a button.

**Comment by Annolago** [comment]
Well, to me a program should have a "nice/clean" way to exit. The "X" in the corner is for hanged programs for most people. 
Anyway, there were/is an exit button in the old program so I hope you see why a user could me forgiven for being somewhat puzzled over the change in behavior. I, at least, waited for a button to show up. When it did not I opened the task Manager just to be certain.
In any case thanks for doing the new version and keep up the good work..
